src/builders.ts

```diff
@@ -453,48 +469,48 @@ export class BomBuilder {
     }
     // endregion fix normalizations
 
-    const component = this.componentBuilder.makeComponent(
+    const newComponent = this.componentBuilder.makeComponent(
```

Comment:
why was this renamed?

Reply:
as the private is named make_compnent i think it would be a better practice to name it as new component as component may works for the first time but in the future it should be better with newcomponent

Reply:
i disagree. please revert this unexpected change.

Reply:
sure